src/server.py

```diff
@@ -34,22 +38,23 @@ def server():
                 message_complete = True
 
         try:
-            parse_request(recv_message.decode('utf-8'))
+            client_uri = parse_request(recv_message.decode('utf-8'))
         except TypeError as msg:
             reply = response_error(405, str(msg))
         except ValueError as msg:
             reply = response_error(400, str(msg))
         else:
-            reply = response_ok()
-
+            resolved = resolve_uri(client_uri)
```

Comment:
Is it possible that this might raise an exception?  What if there's nothing where the requested URI is looking?